parse/PythonParser.cpp

```diff
@@ -153,6 +155,15 @@ PythonParser::PythonParser(PythonCommon& _python, const boost::filesystem::path&
             .def(py::self_ns::self + std::string())
             .def(std::string() + py::self_ns::self);
         py::class_<value_ref_wrapper<Visibility>>("ValueRefVisibility", py::no_init);
+        py::class_<value_placeholder>("ValuePlaceHolder", py::no_init)
+            .def("__call__", &value_placeholder::call<double>)
+            .def(py::self_ns::self + double())
+            .def(py::self_ns::self * double())
+            .def(py::self_ns::self / double())
+            .def(py::self_ns::self - int())
+            .def(int() - py::self_ns::self)
+            .def(py::self_ns::self * int())
+            .def(py::self_ns::self / int());
```

Comment:
Should we just implement all 16 operations?

`(+ - / *) X (int, float) X {direct, reverse}`

A bit of boilerplate but will save time in the future. 

If it's possible to put this into a separate file, I can write a script that will generate it.  Or maybe C++ has some metaprogramming features to do it.

Reply:
Yes, we should but I have no idea how to declare it.